mod/research/set/self/research_page/params.rb

```diff
@@ -74,7 +74,7 @@ def research_param_keys
     if project?
       keys << :project
     else
-      keys += %i[metric_id_list company_id_list year_id_list]
+      keys += %i[metric_id_list company_id_list] # year_id_list]
```

Comment:
you can probably remove it if there are no values in year_id_list but removing it always seems like a bad idea to me

Reply:
I looked for and failed to find a case where this was useful. Can you explain why you'd ever want it?

It creates a lot of noise and makes links all but unsharable.  And it looks like it just gets populated for no good reason.

Reply:
I think there are additional parameters now for all the cases how we currently access the research page (project or one element pinned) so that we probably don't really need any of these id lists but I don't want to cripple an api in one place and leave code in a lot of other place that doesn't work anymore. I made a PR to your branch to make less noisy urls.